README.rst

```diff
@@ -0,0 +1,54 @@
+Scality Object Storage backend for OpenStack Glance
+===================================================
+This package implements a back-end for OpenStack_ Glance_ storing objects in a
+Scality_ RING installation.
+
+.. _OpenStack: http://openstack.org
+.. _Glance: http://docs.openstack.org/developer/glance/
+.. _Scality: http://scality.com
+
+Installation
+------------
+This package depends on `Scality Sproxyd client`_, a Python client library for Scality Sproxyd connector. It must 
+be installed first before installing this package.
+
+.. _Scality Sproxyd client: https://github.com/scality/scality-sproxyd-client
+
+1. Install this package:
+
+   .. code-block:: console
+
+       python setup.py install
+
+2. Configure Glance API. Edit your *glance-api.conf* file to add the Scality backend to the list of known
+   Glance stores:
+
+   .. code-block:: ini
+
+    [glance_store]
+    stores = glance.store.filesystem.Store,
+             glance.store.scality.Store,
+             glance.store.http.Store
+   **N.B** This is an configuration example. Just make sure *glance.store.scality.Store* appears in the list
+   of *stores* in the *[glance_store]* section of *glance-api.conf*.
+
+3. Set to which Sproxyd connectors to connect to. Edit your *glance-api.conf* file and add, in the 
```

Comment:
Remove 'your' and 'file'. Move the section information to the end of the sentence so there's no sub-sentence.